spec/models/hotel_spec.rb

```diff
@@ -3,5 +3,28 @@
 RSpec.describe Hotel, type: :model do
   describe "relationships" do
     it { should have_many :rooms }
+    it { should have_many :reservations }
```

Comment:
There should be 2 relationship tests here: `it { should have_many :reservations}` AND `it { should have_many(:guests).through(:reservations) }`